cfgov/unprocessed/apps/prepaid-agreements/css/main.less

```diff
@@ -0,0 +1,119 @@
+@import (reference) '../../../css/main.less';
+@import (less) '../../../css/search.less';
+@import (less) './search-bar.less';
+
+.u-readable-width {
+    max-width: 41.875rem;
```

Comment:
Add comment that this magic number matches the width set in cf-layout [here](https://github.com/cfpb/capital-framework/blob/0b15295c00ba1972676b6f4c86f3b40dcf45289d/packages/cf-layout/src/cf-layout.less#L252).

Reply:
Good point -- updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96. One thought I had in adding this is that it would be useful to set this value as a variable in cf-layout and/or move the utility class to cf-utility -- we use the number multiple times in various cfgov-refresh less files.